lib/resources/subscriptions.js

```diff
@@ -305,7 +305,7 @@ module.exports = function subscriptionsApi (api) {
       }, normalizeNotes(notes))
 
       return api.post({
-        url: 'subscription_registration/auth_links',
+        url: '/subscription_registration/auth_links',
```

Comment:
Please test this before merging if double slash goes in request, it might fail

Reply:
Yes, I checked this and there is no double slash. 

attach url screenshot 
<img width="369" alt="Screenshot 2023-06-23 at 4 27 46 PM" src="https://github.com/razorpay/razorpay-node/assets/89454448/d83843cf-5b5c-4c51-8bfd-844628e7b372">